### PR TITLE
[WIP] Central error handling

### PR DESCRIPTION
**Pull request by vvmruder**

### CODE REVIEW

File: pyramid_oereb/views/webservice.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def get_extract_by_id(self):
                 raise HTTPNoContent()
             except NotImplementedError:
                 raise HTTPServerError()
-            except NoResultFound:
+            except IOError:
```
[the remark this answers is not in the document]
Comment (sbrunner) [reply pyramid_oereb/views/webservice.py]: +1

**Comment by kdeininger** [comment]
I think we can close this PR as it is far behind the current master. Do you agree?
@vvmruder @sbrunner

**Comment by voisardf** [comment]
I thought the same yesterday... 👍 or we do an overhaul...

**Comment by sbrunner** [comment]
I don't know, I think that's useful to have more informations in the logs :-)

**Pull request closed without merging**